modules/installation-openshift-local.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * getting_started/openshift-overview.adoc
+// * installing/index.adoc
+
+:_content-type: CONCEPT
+[id="installation-openshift-local_{context}"]
+= {product-title} Local Overview
+
+Red Hat {product-title} Local (formerly Red Hat CodeReady Containers) supports rapid application development to get started building {product-title} clusters. {product-title} Local is designed to run on a local computer to simplify setup and testing, and to emulate the cloud development environment locally with all of the tools needed to develop container-based applications. 
```

Comment:
Not sure if you still need to reference `(formerly Red Hat CodeReady Containers)`